src/components/Editor/index.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import { Editor } from "react-draft-wysiwyg";
+import { Container } from './styles';
+import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
+
+const TextEditor = () => {
+    const [editorState, setEditorState] = useState("");
+    const handleEdit = (val) => {
+        setEditorState(val);
```

Comment:
`setEditor` is a function already, why do we need a wrapper over here?
the wrapper does nothing but to pass the params directly to `setEditor`
We should pass the setter function directly in this case.